ee-9/feature-pack/pom.xml

```diff
@@ -475,7 +476,7 @@
                             </task-properties>
                             <jakarta-transform>true</jakarta-transform>
                             <!--<jakarta-transform-configs-dir>EXTERNAL_FOLDER</jakarta-transform-configs-dir>-->
-                            <!--<jakarta-transform-verbose>true</jakarta-transform-verbose>-->
+                            <jakarta-transform-verbose>true</jakarta-transform-verbose>
```

Comment:
Please comment this out again. :)

Reply:
Good idea to use false.  Once this is in we should change it to use a property so people can just set it with -D.

Reply:
damned I thought i just did it ;)